test/samples/typeArray/input.json

```diff
@@ -1,22 +1,26 @@
 {
-	"id": "http://yourdomain.com/schemas/myschema.json",
-	"description": "Example description",
-	"type": "object",
-	"properties": {
-		"first_name": {
-			"type": [
-				"string",
-				"null"
-			]
-		},
-		"last_name": {
-			"type": [
-				"string",
-				"null"
-			]
-		}
-	},
-	"required": [
-		"first_name"
-	]
+    "id": "http://yourdomain.com/schemas/myschema.json",
+    "description": "Example description",
+    "allOf": [
```

Comment:
We dont need to add allOf here as we have other examples that do this, can we revert this please.